devise-otp.gemspec

```diff
@@ -10,6 +10,7 @@ Gem::Specification.new do |gem|
   gem.description = "OTP authentication for Devise"
   gem.summary = "Time Based OTP/rfc6238 compatible authentication for Devise"
   gem.homepage = "https://github.com/wmlele/devise-otp"
+  gem.license = 'MIT'
```

Comment:
It's a small thing, but let's keep strings with " (double quote) so it's all same.